utility_network/configure-subnetwork-trace/src/main/resources/configure_subnetwork_trace/style.css

```
@@ -16,33 +17,39 @@ HBox {
   -fx-font-weight: normal;
 }
 
-
 TextArea {
   -fx-min-height: 181;
   -fx-min-width: 630;
 }
 
-ComboBox {
-  -fx-min-width: 175;
-  -fx-max-width: 175;
+.add-button {
+  -fx-min-width: 40;
 }
 
-TextField {
-  -fx-min-width: 175;
+.trace-button {
+  -fx-min-width: 545;
 }
 
-.comparison-choice-element {
+.reset-button {
   -fx-min-width: 175;
 }
 
-.add-button {
-  -fx-min-width: 40;
+.comparison-sources-combobox {
+  -fx-min-width: 240;
+  -fx-max-width: 240;
 }
 
-.trace-button {
-  -fx-min-width: 545;
+.comparison-operators-combobox {
```

I _think_ it might not be necessary to create a 'styleClass' here, and we could target these using the `fx:id`:
```suggestion
#comparisonOperatorsComboBox {
```

Same would go for the above.

It looks like both parameters are the same for these comboboxes, so it might be possible to create one 'styleClass', or to just do
```
ComboBox{
  -fx-min-width: 240;
  -fx-max-width: 240;
}
```
but keep the extra rule for `.comparison-choice-elements ComboBox`. 
I think in CSS the more specific rule overrides the generic one, so this might be worth trying

---

So had to go with using the `fxid` for those 2, as the `ComboBox` was overriding the other element in `comparison-choice-elements`. But it is still much tidier now!